SumUp/SumUp.h

```diff
@@ -0,0 +1,17 @@
+//
+//  SumUp.h
+//  SumUp
+//
+
+#import <UIKit/UIKit.h>
+
+#import "SumupSDK.h"
+#import "SMPCheckoutRequest.h"
+#import "SMPCheckoutResult.h"
+#import "SMPMerchant.h"
+
+//! Project version number for SumUp.
+FOUNDATION_EXPORT double SumUpVersionNumber;
+
+//! Project version string for SumUp.
+FOUNDATION_EXPORT const unsigned char SumUpVersionString[];
```

Comment:
Where are these provided?

Reply:
The `SumUp.h` header file is automatically generated when you create a Cocoa Touch Framework. It allows you to specify the headers that should be included by the framework. This wrapper framework header should contain the public headers of the original SumUp framework.

The `...VersionNumber` and `...VersionString[]` constants are part of the standard Cocoa Touch Framework template. I believe that the `SumUpVersionNumber` will automatically be filled with the version specified under Build Settings > Versioning > Current Project Version.